versioned_docs/version-5.2/general-concepts/forms-fields/standard-fields/sql.md

```diff
@@ -427,8 +419,34 @@ To **redisplay the form** you can either send the form as an HTTP response to th
 
 or use the Post/Request/Get pattern to redirect to the DisplayController.
 
-You can download [this com_sqlfield component](./_assets/com_sqlfield.zip) as an example to follow (you may need to change the sql_default_catid attribute). 
-Once you have installed the com_sqlfield component you can run the form by navigating to your Joomla instance's site page index.php/component/sqlfield/
+(The example component below uses the first approach).
+
+### Example Component
+
+You can download [this com_sqlfield component](./_assets/com_sqlfield.zip) as an example to follow.
+
+Go onto your Joomla instance back-end, display the article categories and find the id of your `Uncategorised` category.
+
+If it's not 2 then edit the com_sqlfield site/forms/example_form.xml and set the `sql_default_catid` to whatever id it is.
+
+Install the com_sqlfield component and run the form by navigating to your Joomla instance's site page index.php/component/sqlfield/
```

Comment:
```suggestion
Install the com_sqlfield component and run the form by navigating to your Joomla instance's site page `index.php?option=sqlfield`
```

Reply:
Yes, I'll fix that where it occurs (needs to be option=com_sqlfield, of course), and add about using the other only if you have SEF URLs on.

Reply:
don't make it complicated, I think here it's only relevant that you open the component so you don't need to care about sef, in my opinion. The documentation should be straightforward when it's possible without to much "sidenotes" the developer doesn't about in this moment.

we should try to stay with the KISS principals, at least my opinion.